widget/ui/src/containers/SwapInput/SwapInput.tsx

```diff
@@ -5,21 +5,26 @@ import React from 'react';
 
 import { Divider, PriceImpact, Skeleton, Typography } from '../../components';
 
-import { Container, InputAmount, MaxButton } from './SwapInput.styles';
+import {
+  Container,
+  InputAmount,
+  MaxButton,
```

Comment:
Could we improve max button color for the dark themes? (The text is not visible) I've compared it with previous version and it seems to be better.
![Recording 2023-11-15 at 11 24 01](https://github.com/rango-exchange/rango-client/assets/99180934/07dc65a7-be79-4c20-9110-50341500ffe5)

Reply:
fixed